extensions/runtime-version-scanner/app.R

```diff
@@ -53,53 +54,65 @@ ui <- page_sidebar(
     open = TRUE,
     width = 275,
 
-    h5(
-      "Scan for Runtimes",
+    title = "Filters",
+
+    div(
+      "Runtimes",
       tooltip(
         bsicons::bs_icon("question-circle-fill"),
         tagList(
           p(
             "Enable a runtime filter and select a version to show ",
-            "items using that version or earlier.",
+            "items using versions older than the selected version.",
           ),
-          p(
-            "Versions in italic are no longer available on the server."
-          )
         )
       )
     ),
 
```

Comment:
Feel free to defer this idea, but you could add a button that is like "show EOL runtimes only" that sets the filter values to the right things (EOL python, out of tidyverse support for R), as a shortcut to get there.

Reply:
I definitely think a "proper EOL runtimes" filter mode is a good addition for a v1 release! I'm gonna target it in a follow-up PR for the end of the week.